Vesper-Agents-Suite/loss-reported-agent/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "forta-agent-starter",
```

Comment:
from `forta-agent@0.0.19` on the name should be the `agentId`

PD: We should change the publish workflow accordingly